src/binder/bind/bind_ddl.cpp

```diff
@@ -20,45 +20,129 @@ using namespace kuzu::catalog;
 namespace kuzu {
 namespace binder {
 
-std::unique_ptr<BoundStatement> Binder::bindCreateNodeTableClause(
-    const parser::Statement& statement) {
-    auto& createTableClause = reinterpret_cast<const CreateTableClause&>(statement);
-    auto tableName = createTableClause.getTableName();
-    if (catalog.getReadOnlyVersion()->containTable(tableName)) {
-        throw BinderException("Node " + tableName + " already exists.");
+static std::unique_ptr<LogicalType> bindDataType(const std::string& dataType) {
```

Comment:
Why should we return a copy of the input dataType?
I think this function is only validating the FIX_LIST type